README.md

```diff
@@ -0,0 +1,137 @@
+                      EFM Awesome Books Hub
+                                  
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [🚀 Live Demo ](#-live-demo-)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 HTML/CSS & JavaScript capstone project - Conference page<a name="about-project"></a>
+
+-EFM Awesome Books Hub - The  Awesome Books page is my first Microverse  Awesome Books project using ES6 syntaxes and modules. In this project, we use HTML, CSS, and JavaScript and ES6 Modules by adding an index, JS and CSS file, I also used linters to analyze our source code to flag programming errors, bugs, stylistic errors, and suspicious constructs.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.spec.whatwg.org/">HTML</a></li>
+    <li><a href="https://www.w3.org/TR/CSS/#css">CSS</a></li>
+    <li><a href="#">JavaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Set up the linter for the project 👌
+- Add the node modules to the .gitignore file🚀
+- Add the styling
+- Use DOM Manipulation
+- Add modules using ES6 Syntaxes
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+
+1. Computer
+2. Internet
+3. Web Browser
+4. Git
+5. Code Editor
+
+### Setup
+
+Clone this repository to your desired folder Enter the following commands in the terminal:
+
+```sh
+  git clone git@github.com:efmomoh/my-books.git
+  cd awesome-books
+```
+
+### Install
+
+Install this project with the following command in the terminal:
+
+```sh
+ npm install 
+
+```
+
+### Usage
+
+To run the project,
+
+```sh
+  Open the browser and enter the address of the index.html file from your computer/server
+```
+
+### Run tests
+
+```sh
+   1. npx hint.
+   2. npx stylelint "**/*.{css,scss}"
+```
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ - [Live Demo Link](https://efmomoh.github.io/my-books/)
```

Comment:
- [ ] @efmomoh Your live demo link is not working, kindly ensure you have a live demo, you have an option of removing this section until a time you have a live demo.